hal_riscv64/src/mm/mod.rs

```diff
@@ -50,3 +50,7 @@ unsafe fn load_pagetable(pt: &'static mut PageTable) {
         asm!("sfence.vma");
     }
 }
+
+pub fn align_up(addr: usize) -> usize {
+   mm::align_up(addr, PageTable::PAGE_SIZE)
+}
```

Comment:
I think in the future we should just merge hal_core with hal_archXXX with conditional compilation. There is place for some code factoring but that for another time